.github/workflows/cd.yml

```diff
@@ -0,0 +1,15 @@
+name: publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    name: publish to expo
+    uses: nearform/optic-expo/.github/workflows/expo-publish.yml@chore/expo-preview
+    with:
+      channel: default
```

Comment:
minor: shall we make this the default in the reusable workflow so if it's not specified it goes there?